isis/src/mro/apps/hical/main.cpp

```diff
@@ -369,7 +369,7 @@ void IsisMain(){
     HiHistory GucHist;
     GucHist.add("Profile["+ hiprof.Name()+"]");
     if ( !SkipModule(hiprof) ) {
-      GainUnitConversion guc(hiconf, units);
+      GainUnitConversion guc(hiconf, units, hifrom);
```

Comment:
what's the significance of this change?

Reply:
The call to get the sun distance is down a few layers that do not have access to the camera via the cube. So I had to pass it dow